### DISCUSSION TITLE
Tailwind css classes not working when code passed through innerHTML

### DISCUSSION THREAD

**Discussion by sc0rp10n-py** [tailwindlabs/tailwindcss#7482] [Help]
My js code
```js
const header = document.querySelectorAll('.header');
console.log(header);
header.forEach(header => {
    header.innerHTML = `<div class="container mx-auto flex flex-row justify-between items-center"><div>
    <img src="./assets/logo.png" alt="Bull N' Bit" class="w-28 h-14">
</div>
<div class="text-2xl space-x-10 font-semibold">
    <a href="">why mining</a>
    <a href="">profits</a>
    <a href="">about us</a>
    <a href="">
        <button
            class="rounded-lg bg-yellow-600 font-semibold py-3 px-6"
        >
            Contact Us
        </button>
    </a>
</div></div>`;
});
```

All these classes written come up with empty styles on DOM.
Does anyone know the reason behind it?

**Comment by dotnetCarpenter** [comment]
It seems that you are not watching that file with Tailwind CSS.

Either [get all Tailwind CSS](https://tailwindcss.com/docs/installation/play-cdn) with `<script src="https://cdn.tailwindcss.com"></script>`
or [setup _tailwind.config.js_ to watch your files](https://tailwindcss.com/docs/installation) with
```js
module.exports = {
	content: ["./src/**/*.{html,js}"],
	theme: {	
		extend: {},
	},
	plugins: [],
}
```

I recommend the latter.

**Reply by dotnetCarpenter** [reply]
@sc0rp10n-py The documentation at https://tailwindcss.com/docs/content-configuration is probably more instructive.

**Reply by imhilla** [reply]
This really helped today had my components named like so ./Components it should be ./componets on app.config file
